### PR TITLE
basic e2e test for instantiating All Reviews block

### PR DESCRIPTION
**Pull request by haszari**
Part of #2395

This PR adds a basic e2e test for instantiating an All Reviews block. Note the code was pulled directly from #2756 PR, splitting those tests off into separate PRs (divide and conquer).

Credit to @senadir for all the code in this PR :)

### How to test the changes in this Pull Request:

1. Clone this branch.
2. Build e2e container infra stuff `npm run build:e2e-test`
3. Start wp-env containers - tests are run against the wp tests container `npm run wp-env start`
4. Run e2e tests `npm run test:e2e`

Tests should all pass, in particular new `All Reviews` test should pass.

Also the travis tests on this PR should succeed and include the `All Reviews` block test.

### CODE REVIEW

File: tests/e2e-tests/specs/backend/all-reviews.test.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+const block = {
+	name: 'All Reviews',
+	slug: 'woocommerce/all-reviews',
+	class: '.wc-block-all-reviews',
+};
+
+if ( process.env.WP_VERSION < 5.3 )
+	// eslint-disable-next-line jest/no-focused-tests
+	test.only( 'skipping all other things', () => {} );
```
Comment (haszari) [inline tests/e2e-tests/specs/backend/all-reviews.test.js:16]: Not sure what this is doing - is this needed?

Comment (mikejolley) [reply tests/e2e-tests/specs/backend/all-reviews.test.js:16]: This disables the tests on old versions of WP where the block is not supported (?).

Comment (mikejolley) [reply tests/e2e-tests/specs/backend/all-reviews.test.js:16]: Should be removed for the reviews block.

Comment (haszari) [reply tests/e2e-tests/specs/backend/all-reviews.test.js]: > This disables the tests on old versions of WP where the block is not supported (?).

Aha! That makes sense. Let's change that test name/string on the other block tests so it's clearer :)

**Pull request merged**